### PR TITLE
Add missing and needed arguments with the new api

### PR DESCRIPTION
**Pull request by vrseraphin**
Adopt new loadDefaultPortal api

### CODE REVIEW

File: src/samples/FullPortal/index.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default function FullPortal() {
     else if( myLoadDefaultPortal ) {
       // eslint-disable-next-line no-console
       console.log(`Loading default portal`);
-      myLoadDefaultPortal();
+      myLoadDefaultPortal("pega-root", []);
```
Comment (JEAxero) [inline src/samples/FullPortal/index.tsx:139]: @vrseraphin  A few questions:

* Will this change break 8.7? (the main branch currently works for both 8.7 and 8.8)
* Will this change also be made to SDK-A and SDK-WC soon so they stay in sync?

Comment (vrseraphin) [reply src/samples/FullPortal/index.tsx:139]: No.  As myLoadDefaultPortal will be undefined and this code path will not be traversed.  (so a capability check.  Will use only if there

**Review by vishalshrm539 — APPROVED** [review]

File: src/samples/FullPortal/index.tsx (hunk 1 above)
Comment (vrseraphin) [reply src/samples/FullPortal/index.tsx:139]: Yes, will get changes into other SDKs as well.

**Review by JEAxero — APPROVED** [review]
Ran tests. Looks good.

**Pull request merged**